util/discord.js

```diff
@@ -30,6 +40,39 @@ module.exports = async function (args, context = epochtal) {
       await discordClient.channels.cache.get(context.data.discord.announce).send({ content, files });
       return "SUCCESS";
 
+    case "releasemap":
+
+      // Download map thumbnail
+      let thumbnail = context.data.week.map.thumbnail;
+      if (!thumbnail.startsWith("http")) {
+        thumbnail = `https://steamuserimages-a.akamaihd.net/ugc/${thumbnail}?impolicy=Letterbox&imh=360`;
+      }
+
+      const response = await fetch(thumbnail);
+      const buffer = await response.arrayBuffer();
+
+      // Render map release image
+      const image = await render(
+        buffer,
+        context.data.week.map.title,
+        context.data.week.map.author,
+        context.data.week.map.upvotes,
+        context.data.week.map.downvotes
+      );
+
+      const imagePath = (await tmppath()) + ".png";
+      Bun.write(imagePath, image);
```

Comment:
does this ever get cleaned up?

Reply:
(also that's an async write without an await!!)

Reply:
how about 7 lines below?

Reply:
nvm, TIL `unlink` isn't just for symlinks

Reply:
> (also that's an async write without an await!!)
this is why we need eslint, lmfao

Reply:
nah thats just an excuse to write bad code :Kappa: